### ISSUE TITLE
`pad_conditioning_tensors_to_same_length()` fails if prompts have different non-truncated chunked lengths

### ISSUE THREAD

**Issue by abassino** [damian0815/compel#18]
Hello and thank you for the amazing work. 
I am currently testing the last version (1.0.5) and it works well when the embedding of a list of prompts are of the same length after using `compel.pad_conditioning_tensors_to_same_length`. But it fails if the embedding are of different length, for example with a short prompt and a long prompt. 
Below is the snippet of the test code and the generated error:
```python
import torch
from diffusers import StableDiffusionPipeline
from compel import Compel

pipeline = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16, revision="fp16").to("cuda")
pipeline.enable_xformers_memory_efficient_attention()
compel = Compel(tokenizer=pipeline.tokenizer, text_encoder=pipeline.text_encoder, truncate_long_prompts=False)

prompts = ["a dog", "a cat playing with a ball++ in the forest, amazing, exquisite, stunning, masterpiece, skilled, powerful, incredible, amazing, trending on gregstation, greg, greggy, greggs greggson, greggy mcgregface, tyndall effect,photorealistic, dark studio, rim lighting, two tone lighting,(high detailed skin), 8k uhd, dslr, soft lighting, high quality, volumetric lighting, candid, Photograph, high resolution, 4k, 8k, Bokeh"]
negative_prompts = ["blur", "blur"]
prompt_embeds = compel(prompts)
negative_prompt_embeds = compel(negative_prompts)

[prompt_embeds, negative_prompt_embeds] = compel.pad_conditioning_tensors_to_same_length([prompt_embeds, negative_prompt_embeds])

images = pipeline(prompt_embeds=prompt_embeds, negative_prompt_embeds=negative_prompt_embeds).images

images[0].save("image0.jpg")
images[1].save("image1.jpg")
````

The corresponding error is:

```cmd
RuntimeError: Sizes of tensors must match except in dimension 0. Expected size 77 but got size 154 for tensor number 1 in the list.
```
Thank you in advance!

**Comment by damian0815** [comment]
yeah i guess i didn't test that use case. i'll take a look.

**Comment by alexisrolland** [comment]
Hello @damian0815, I'm upgrading compel to 1.1.5 and I see this issue is still present. So for now I'm planning to deactivate  `truncate_long_prompts=False`. I would like to know what happens to the prompt / images when `truncate_long_prompts=True`?

Thank you

**Comment by damian0815** [comment]
fixed in 248bcb6de3f0c4cf717cfb430e669b9b6f1d594f
